static_src/stores/user_store.js

```diff
@@ -164,6 +150,7 @@ export class UserStore extends BaseStore {
         const orgPermissionsReq = cfApi.deleteOrgUserPermissions(
           action.userGuid,
           action.orgGuid,
+          'users',
```

Comment:
it's meant to be duplicated but let me re-work it so it's more clear as to why it's like that.